src/pages/User/utils/getUserById.ts

```diff
@@ -0,0 +1,31 @@
+import { User } from 'src/utils';
+import { useRoute, useRouter } from 'vue-router';
+import { getSpecificUser } from 'src/actions';
+import { ref } from 'vue';
+import { Notify } from 'quasar';
+
+export const getUserbyId = async () => {
+  const user = ref<User>();
+  const route = useRoute();
+  const id = 'b5ca33ca-fd87-4858-a285-e15429c6dd9b';
```

Comment:
Please add "id" as an parameter, maybe you can add the value as a "parameter default", and we need to create an issue to remove that default when Authentication was implemented

```suggestion
export const getUserbyId = async (id = 'b5ca33ca-fd87-4858-a285-e15429c6dd9b') => {
  const user = ref<User>();
  const route = useRoute();
```